packages/access-token/package.json

```
@@ -12,6 +12,7 @@
     "test": "ava",
     "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
     "test:xs": "exit 0",
+    "lint": "yarn lint:eslint",
```

consider defining this in a way that doesn't need updating when a new linter is added.

```suggestion
    "lint": "run-s --continue-on-error lint:*",
```